glide.yaml

```diff
@@ -0,0 +1,18 @@
+package: .
+homepage: https://github.com/HewlettPackard/oneview-golang
+import:
+- package: github.com/docker/machine
+  version: v0.8.0-rc2
+  subpackages:
+  - libmachine/log
+- package: github.com/stretchr/testify
+  version: v1.1.3
+  subpackages:
+  - assert
+excludeDirs:
```

Comment:
I'm curious why the `excludeDirs` usage? For example, I see `rest` only imports `utils` and `testify` (which is elsewhere). I'm curious of your motivation.